index.js

```diff
@@ -85,10 +86,11 @@ function trim(str) {
   return replace.call(str, trimRe, '');
 }
 
+var dashRe = /-/;
 function pluralTypeName(locale) {
   var langToPluralType = langToTypeMap(pluralTypeToLanguages);
   return langToPluralType[locale]
-    || langToPluralType[locale.split(/-/, 1)[0]]
+    || langToPluralType[split.call(locale, dashRe, 1)[0]]
```

Comment:
actually in this case, because it's not a global regex, it's better to recreate the regex inline - otherwise its `lastIndex` will persist across calls.